src/pages/teams.astro

```diff
@@ -0,0 +1,363 @@
+                your needs and feedback as a beta user.</span
+              >
+              <a class="button button-blue mt-8">
+                <span> Send Feeback</span>
+              </a>
+            </div>
+          </div>
+          <div
+            class="border-r border-b border-l border-gray-500 rounded-b-[40px] pt-10 pb-16"
+          >
+            <span
+              class="mt-1 text-xs font-medium tracking-widest sm:tracking-[0.2em] text-gray-400"
+              >SOME OF THE FEATURES WE’RE CONSIDERING</span
+            >
+            <div
+              class="m-auto max-w-2xl grid grid-cols-2 sm:grid-cols-4 gap-y-5 mt-7 sm:mt-5"
+            >
+              <div class="flex flex-col text-gray-400">
+                <Icon icon="user-add-line text-xl text-blue-500" />
+                <span class="mt-1 text-sm text-gray-200">Single Sign On</span>
+              </div>
+              <div class="flex flex-col text-gray-400">
+                <Icon icon="remote-control-2-line text-xl text-brand-pink" />
+                <span class="mt-1 text-sm text-gray-200">Access Controls</span>
+              </div>
+              <div class="flex flex-col text-gray-400">
+                <Icon icon="calendar-2-line text-xl text-green-500" />
+                <span class="mt-1 text-sm text-gray-200">Scheduling</span>
+              </div>
+              <div class="flex flex-col text-gray-400">
+                <Icon icon="file-list-2-line text-xl text-red-600" />
+                <span class="mt-1 text-sm text-gray-200">Audit Log</span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section id="faq" class="bg-gray-900">
+    <div class="m-auto max-w-7xl px-4 py-16 flex flex-col items-center">
+      <h2 class="text-gray-50 text-2xl md:text-4xl text-center">
+        Frenquently Asked Questions
+      </h2>
+
+      <div
+        class="mx-auto md:w-[80%] lg:w-[45%] mt-10 flex flex-col justify-start space-y-6"
+      >
+        <!--Is Livebook Teams Free to use? -->
+        <div class="space-y-2">
+          <a class="faq-question-btn flex justify-between cursor-pointer">
+            <h3 class="text-gray-200">Is Livebook Teams Free to use?</h3>
+            <Icon
+              icon="arrow-drop-down-line text-xl text-gray-200"
+              class="hidden faqIconDown"
+            />
+            <Icon
+              icon="arrow-drop-up-line text-xl text-gray-200"
+              class="faqIconUp"
+            />
+          </a>
+          <span class="faq-answer space-y-3 text-sm mt-1 text-gray-400"
+            >Livebook Teams is not only free but also open source, providing you
+            with a powerful collaborative platform at no cost. We believe in the
+            importance of accessible tools for knowledge sharing and
+            collaboration</span
+          >
+        </div>
+        <hr class="border-gray-800" />
```

Comment:
Now that we use astro we can easily generate content, such as a list of entries, and this way we don't have to repeat the markup.

At the top of the file you can define this:

```js
const faqItems = [
  {
    question: "Is Livebook Teams Free to use?",
    answer: "..."
  },
  ...
];
```

Then here you just loop over the items:

```jsx
{faqItems.map((item) =>
  <div>
    ...
  </div>
)}
```

You can inject the content using `{item.answer}` and `{item.question}`.

Reply:
Very cool feature. I liked it! =)

Just one question: the first question of the FAQ is different from the rest, because is visible by default.

![Screenshot 2023-06-20 at 09 22 49](https://github.com/livebook-dev/livebook.dev/assets/6081301/adc45822-8626-4117-a6e6-f97131a35aa9)

In this case, should I do something like this?

` <div
        class="mx-auto md:w-[80%] lg:w-[45%] mt-10 flex flex-col justify-start space-y-6"
      >
        <div class="space-y-2">
          <a
            class="faq-question-btn flex justify-between cursor-pointer text-gray-200"
          >
            <h3>Is Livebook Teams Free to use?</h3>
            <Icon
              icon="arrow-drop-down-line"
              class="text-xl hidden faqIconDown"
            />
            <Icon icon="arrow-drop-up-line" class="text-xl faqIconUp" />
          </a>
          <span class="faq-answer space-y-3 text-sm mt-1 text-gray-400"
            >Livebook Teams is not only free but also open source, providing you
            with a powerful collaborative platform at no cost. We believe in the
            importance of accessible tools for knowledge sharing and
            collaboration</span
          >
        </div>
        <hr class="border-gray-800" />

        {
          faqItems.map((item) => (
            <>
              <div class="space-y-2">
                <a class="faq-question-btn flex justify-between cursor-pointer text-gray-200">
                  <h3>{item.question}</h3>
                  <Icon
                    icon="arrow-drop-down-line hidden"
                    class="text-xl faqIconDown"
                  />
                  <Icon icon="arrow-drop-up-line" class="text-xl faqIconUp" />
                </a>
                <span class="hidden faq-answer space-y-3 text-sm mt-1 text-gray-400">
                  {item.answer}
                </span>
              </div>
              <hr class="border-gray-800" />
            </>
          ))
        }
      </div>`

Reply:
Not much =/, but I'll take a look, and if I cannot figure it out, I'll text you on Slack =)